c++/src/Timezone.cc

```diff
@@ -675,6 +675,13 @@ namespace orc {
     if (itr != timezoneCache.end()) {
       return *(itr->second).get();
     }
+    if (!std::filesystem::exists(std::filesystem::path(filename))) {
+      std::stringstream ss;
+      ss << "Time zone file " << filename << " does not exist."
+         << " Please install IANA time zone database and set TZDIR env properly"
```

Comment:
Just a suggestion: Can rephrase as 

"Please install IANA time zone database and set TZDIR env property to point to /usr/share/zoneinfo" or something like that

Reply:
What about this: `Please install IANA time zone database and set TZDIR env if it is not installed at /usr/share/zoneinfo`